app/scripts/controllers/error.ctrl.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('ErrorCtrl', function (error) {
+  var ctrl = this;
+  ctrl.errorCode = (error)?error.code:"";
```

Comment:
parenthesis here don't add anything. Also, please space this out for better readability.